src/components/CGroupContent.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <tr :key="groupId + 0">
+    <td
+      class="border text-xs px-4 py-2"
```

Comment:
can you add pagination for groups too ? and filters ? don't bother with sort (just sort by enqued datetime)